frontend/tests/playwright/utils.ts

```diff
@@ -22,6 +22,15 @@ export const interceptLogin = async (page: Page, userParams?: Object) => {
    );
 };
 
+export const waitForAlgoliaSearch = async (page: Page) => {
+   return page.waitForResponse(async (response) => {
+      return (
+         response.url().includes('1/indexes/*/queries') &&
```

Comment:
Not super familiar with the algolia requests, but why is there a `1` at the begining? Also, we did we drop the `algolia` check?

Reply:
I am not sure about the `1` either. This is similar to what we do with Cypress 
https://github.com/iFixit/react-commerce/blob/main/frontend/tests/cypress/support/commands.ts#L28